<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mode_time</class>
 <widget class="QWidget" name="mode_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Mypushbutton" name="Start">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="Mypushbutton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert to txt</string>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Insert the desired sampling period</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Time</string>
   </property>
  </widget>
  <widget class="LCDNumber_reverse" name="number">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Mypushbutton</class>
   <extends>QPushButton</extends>
   <header>my_pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>LCDNumber_reverse</class>
   <extends>QLCDNumber</extends>
   <header>lcdnumber_reverse.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
